### ISSUE TITLE
Links to AAR Artifact Broken

### ISSUE THREAD

**Issue by cyon1c** [commonsguy/cwac-anddown#22]
Mark - thanks for a great library! I'm trying to resuscitate a project of mine that utilizes cwac-anddown.

Trying to get rid of my cwac-anddown submodule and use the AAR artifact. I've tried with both http/https to no avail.

Is the information in the Readme still up to date?

**Comment by commonsguy** [comment]
It seems to be.

In a scrap project, I modified the top-level `build.gradle` to have:

```gradle
allprojects {
    repositories {
        google()
        jcenter()
        maven {
            url "https://s3.amazonaws.com/repo.commonsware.com"
        }
    }
}
```

And I modified `app/build.gradle` to include:

```gradle
compile 'com.commonsware.cwac:anddown:0.4.0'
```

(despite the old `compile` instead of the newer `implementation`)

And, the artifact seemed to come down just fine when I sync'd the project. 

I would need more specific symptoms to investigate further (e.g., output from the Gradle Console).
